test/e2e/performanceprofile/functests/1_performance/cpu_management.go

```diff
@@ -939,6 +939,8 @@ var _ = Describe("[rfe_id:27363][performance] CPU Management", Ordered, func() {
 			guaranteedPod = makePod(ctx, workerRTNode, true)
 			err := testclient.Client.Create(ctx, guaranteedPod)
 			Expect(err).ToNot(HaveOccurred(), "Failed to create guaranteed pod")
+			guaranteedPod, err = pods.WaitForCondition(ctx, client.ObjectKeyFromObject(guaranteedPod), corev1.PodReady, corev1.ConditionTrue, 10*time.Minute)
```

Comment:
this commit has 2 different changes, I'd split them because they are not related.
Also let's add commit messages to all commits please